crates/matrix-sdk-sled/src/state_store.rs

```diff
@@ -188,14 +192,23 @@ pub struct SledStateStoreBuilderConfig {
 
 impl SledStateStoreBuilder {
     pub fn path(&mut self, path: PathBuf) {
-        self.path = Some(path.join("matrix-sdk-state"));
+        self.db_or_path = Some(DbOrPath::Path(path.join("matrix-sdk-state")));
```

Comment:
Can we do the path joining inside `build` as before please?

Reply:
Sure, had to change the structure a little bit to not repeat myself.